docs/pages/setup/guides/joining-nodes-aws-ec2.mdx

```diff
@@ -207,6 +212,16 @@ proxy_service:
 
 ## Step 4/4. Launch your Teleport node
 
+<Admonition
+  type="note"
+  title="IMPORTANT"
+>
+  The data directory used (e.g. `/var/lib/teleport` by default) should be empty prior to launching
+  node.  An error will occur if this node had attached by another method, such as IAM and now switching to
+  EC2 Joining. 
+  </Admonition>
```

Comment:
ah this closing tag is unhighlighted in GH because it's indented (and the opener isn't)
```suggestion
</Admonition>
```